controllers/policy_test.go

```diff
@@ -314,4 +315,320 @@ var _ = Describe("ImagePolicy controller", func() {
 			})
 		})
 	})
+
+	Context("Access ImageRepository", func() {
```

Comment:
Hooray for comprehensive tests! Can the body of the tests be factored out?

Reply:
I'll leave this for when we'll refactor all tests and get rid of Ginko.

Reply:
That's a bit of a dodge, but OK if you promise :-)